lib/output.rb

```diff
@@ -62,8 +62,8 @@ def sortuniq(p)
 class OutputObject < Output
   def out(target, status, results)
     $semaphore.synchronize do
-      @f.puts "Identifying: #{target}"
```

Comment:
I went without the `.to_s` due to [this](https://github.com/bbatsov/ruby-style-guide#no-to-s)